.github/workflows/main.yml

```diff
@@ -26,27 +32,3 @@ jobs:
       - name: Test
         run: npm test
 
-  release:
-    needs: test
```

Comment:
I don't see this dependency on `test` in `release.yml`. This means a release could happen with broken tests.

Reply:
My assumption, based on what I read in the docs, is that jobs can not be shared between workflow files, so I removed this bit. Happy to check if I'm wrong here if we think we need to run tests before every release and don't want to have to copy-paste between these workflow files.